packages/expo-development-client/ios/EXDevelopmentClientLoadingView.m

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import "EXDevelopmentClientLoadingView.h"
+
+#if __has_include(<React/RCTDevLoadingViewProtocol.h>)
+
+@implementation EXDevelopmentClientLoadingView
+
++ (NSString *)moduleName
+{
+  return @"RCTDevLoadingView";
+}
+
++ (void)setEnabled:(BOOL)enabled {}
+- (void)showMessage:(NSString *)message color:(UIColor *)color backgroundColor:(UIColor *)backgroundColor {}
```

Comment:
Same here, for prebuilt libs you should add imports to both `Foundation` and `UIKit`